src/app/app.component.ts

```diff
@@ -12,6 +12,8 @@ export class AppComponent {
   storyOptions: NgxStoriesOptions = {
     width: 380,
```

Comment:
Can you please update the width & height to 338 & 600 respectively ?
As it was implemented in app's template only.
And the story view looks good with that particular width and height only.

Reply:
Sure